Fix version

IgorIoT13 opened the pull request.

IgorIoT13 committed:
Now start all relay only with FAN

IgorIoT13 committed:
add debug header

IgorIoT13 committed:
back to normal

IgorIoT13 committed:
add serial _func_

IgorIoT13 committed:
Button fix

IgorIoT13 committed:
Try fix lcd

IgorIoT13 committed:
fix menu lcd

IgorIoT13 committed:
add encoder header

IgorIoT13 committed:
Work Btn and menu + prototype enc

olehdzhufer committed:
adding relay timer for on/off Heat and Cool

olehdzhufer committed:
adding some fixes

IgorIoT13 committed:
work enc

VladVanyuk committed:
Removed copy file of schematic

VladVanyuk committed:
Update menu.h

#include "coreMenu/MenuLib.h"
changed position

VladVanyuk committed:
Update serverHost.h

#include "ESPAsyncWebServer.h"
position

olehdzhufer committed:
update timer

IgorIoT13 committed:
add counterTimer header

IgorIoT13 committed:
add timer to relay

IgorIoT13 committed:
add timer and temp sensor

olehdzhufer committed:
adding class to encoder

IgorIoT13 committed:
Merge branch 'Fix_version' into timer

IgorIoT13 committed:
Merge pull request #17 from olegdzhufer/timer

Timer

IgorIoT13 committed:
Merge branch 'developer_branch' into Fix_version

VladVanyuk commented on Enc.h:
```diff
@@ -0,0 +1,76 @@
+    }
+
+    else if (en.left()) {
+        enc_pre = 0x03;
```
What does this flag do? 
Name them

VladVanyuk replied:
@ @olehdzhufer @olegdzhufer

VladVanyuk commented on btns.h:
```diff
@@ -7,21 +7,35 @@
 #include "relay.h"
 
+
+bool fan = false;
```
fan_active_fl

VladVanyuk replied:
@IgorIoT13 @olehdzhufer @olegdzhufer

VladVanyuk commented on Enc.h:
```diff
@@ -0,0 +1,76 @@
+  return enc_pre; 
+}
+
+bool flagEnc = false;
```
What purpose of this?

VladVanyuk replied:
Same flag as in prev issue? 
@IgorIoT13  @olehdzhufer

VladVanyuk committed:
Update Enc.h

Added debug macros;
Added define for max limit for temperature to set;
Changed serial print to debug print;
Deleted enc_pre;
Deleted encFlag and function getResult();
Comented unused sections;

VladVanyuk commented on debug_helper.h:
```diff
@@ -0,0 +1,28 @@
+
+#define INFO_WIFI
+
+void debugMeseg(char* meseg){
```
Chenge to macros

VladVanyuk replied:
@IgorIoT13

VladVanyuk commented on tempSensor.h:
```diff
@@ -32,6 +35,9 @@ float readSensorComplete()
+    #ifdef DEBUG_FUNC
+      Serial.println(__func__);
+    #endif
 #if(VIRT_SENSOR == 0)
```
delete all examples of this define

VladVanyuk replied:
@olehdzhufer @olegdzhufer

VladVanyuk commented on menu.h:
```diff
@@ -58,17 +62,27 @@ STATUS_t initSection() {
     }
+
+  footerLine = initLine( "Timer: ", "Timer", NULL);
+  footerLine->val->setChar(footerLine->val, "");
```
Double check this functionality. You are parsing a pointer to a char array to the pointer of char

VladVanyuk reviewed (approved):
Change what i said in comments

VladVanyuk commented:
@IgorIoT13 @olehdzhufer @olegdzhufer
